common/messages/messages_en.go

```diff
@@ -42,49 +42,49 @@ var messagesEn = map[string]util.Message{
+		Action:      "Verify that the volume ID exists. Run 'ibmcloud is shares' to list available volumes in your account. If the ID is correct, try to delete the volume with the 'ibmcloud is share-delete' command. ",
 	},
 	"FailedToUpdateVolume": {
 		Code:        "FailedToUpdateVolume",
 		Description: "The volume ID '%d' could not be updated",
 		Type:        util.UpdateFailed,
 		RC:          500,
-		Action:      "Verify that the volume ID exists. Run 'ibmcloud is volumes' to list available volumes in your account.",
+		Action:      "Verify that the volume ID exists. Run 'ibmcloud is shares' to list available volumes in your account.",
 	},
 	"StorageFindFailedWithVolumeId": {
 		Code:        "StorageFindFailedWithVolumeId",
 		Description: "A volume with the specified volume ID '%s' could not be found.",
 		Type:        util.RetrivalFailed,
 		RC:          404,
-		Action:      "Verify that the volume ID exists. Run 'ibmcloud is volumes' to list available volumes in your account.",
+		Action:      "Verify that the volume ID exists. Run 'ibmcloud is shares' to list available volumes in your account.",
 	},
 	"StorageFindFailedWithVolumeName": {
 		Code:        "StorageFindFailedWithVolumeName",
 		Description: "A volume with the specified volume name '%s' does not exist.",
 		Type:        util.RetrivalFailed,
 		RC:          404,
-		Action:      "Verify that the specified volume exists. Run 'ibmcloud is volumes' to list available volumes in your account.",
+		Action:      "Verify that the specified volume exists. Run 'ibmcloud is shares' to list available volumes in your account.",
 	},
 	"AccessPointWithAPIDFindFailed": {
 		Code:        AccessPointWithAPIDFindFailed,
 		Description: "No volume access point could be found for the specified volume ID '%s' and access point ID '%s'",
 		Type:        util.VolumeAccessPointFindFailed,
 		RC:          400,
-		Action:      "Verify that a volume access point for your volume exists.Check if volume ID and access point ID is valid",
+		Action:      "Verify that a volume access point for your volume exists. Run `ibmcloud is share-targets <VOLUME_ID>` to list all volume access point for a file share. Check if volume ID and access point ID is valid",
 	},
 	"AccessPointWithVPCIDFindFailed": {
 		Code:        AccessPointWithVPCIDFindFailed,
 		Description: "No volume access point could be found for the specified volume ID '%s' and VPC ID %s",
 		Type:        util.VolumeAccessPointFindFailed,
 		RC:          400,
-		Action:      "Verify that a volume access point for your volume exists.Check if volume ID and VPC ID is valid",
+		Action:      "Verify that a volume access point for your volume exists. Run `ibmcloud is share-targets <VOLUME_ID>` to list all volume access point for a file share. Check if volume ID and VPC ID is valid",
```

Comment:
Run ibmcloud is share-targets <VOLUME_ID> to list all volume access point for a volume.